l10n/en/navigation.ftl

```diff
@@ -19,9 +19,6 @@ navigation-watch-videos-and-browse = Watch videos and browse the internet on you
 navigation-firefox-browsers-put = { -brand-name-firefox } browsers put your privacy first — and always have.
 navigation-take-the-passwords-youve = Take the passwords you’ve saved in { -brand-name-firefox } with you everywhere.
```

Comment:
These are gone too… FireTV, VR goggles, lockwise… isn't most of the v1 navigation gone already? (I see only bits like the navigation-firefox-blog at the end here being used as a fallback for navigation-v2-firefox-blog e.g., but otherwise this is probably all obsolete kinda implicitly, being removed at some point via nav redesign…)

Reply:
Hmm I just noticed that too. I think I'll create a separate issue for this so this PR doesn't get too large!